src/_cffi_src/openssl/objects.py

```diff
@@ -9,6 +9,14 @@
 """
 
 TYPES = """
+typedef struct {
+    int type;
+    int alias;
+    const char *name;
+    const char *data;
+} OBJ_NAME;
```

Comment:
Okay, looking at the code there's `EVP_MD_do_all` in newer OpenSSL but that just calls `OBJ_NAME_do_all` anyway, so it appears this is indeed required. Okay 👍